src/utils/transaction.ts

```diff
@@ -160,6 +160,113 @@ export const klimaDaoStaking = async (
   }
 };
 
+export const plrDaoStaking = async (
```

Comment:
Snack case please

```suggestion
export const PlrDaoStaking = async (
```

Reply:
hmmm @IAmKio I think this is good cuz it's method name?